Dockerfile

```diff
@@ -12,18 +11,29 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
     curl \
     ca-certificates \
     git
```

Comment:
(Optional) Seems like it would longer be necessary to install `git` and `curl` packages now, as they are installed in the `golang:1.12.5-stretch` image already:

```
sudo docker run --rm -it golang:1.12.5-stretch bash -c 'which curl && which
git'
/usr/bin/curl
/usr/bin/git
```

Reply:
Removed curl and git